### PR TITLE
Update TiFlash related info in quickstart page

### PR DESCRIPTION
**Pull request by TisonKun**
### First-time contributors' checklist

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update TiFlash related info in quickstart page.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

N/A

### Do your changes match any of the following descriptions?

N/A

### CODE REVIEW

**Comment by TomShawn** [comment]
@zanmato1984 Please take a look and see whether this PR also applies to v4.0, thanks!

**Comment by TisonKun** [comment]
> @zanmato1984 Please take a look and see whether this PR also applies to v4.0, thanks!

@TomShawn I come from [this article](https://zhuanlan.zhihu.com/p/150946650). It says tiflash supported after v4.0.1 and I've verified it. Not sure if it means applied to v4.0

**Comment by TomShawn** [comment]
> > @zanmato1984 Please take a look and see whether this PR also applies to v4.0, thanks!
> 
> @TomShawn I come from [this article](https://zhuanlan.zhihu.com/p/150946650). It says tiflash supported after v4.0.1 and I've verified it. Not sure if it means applied to v4.0

I mean v4.0.x, which includes v4.0.0 and v4.0.1.

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -96,7 +96,11 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
 
 6. Access the [TiDB Dashboard](/dashboard/dashboard-intro.md) at <http://127.0.0.1:2379/dashboard>. The default username is `root`, with an empty password.
 
-7. After the test, you can clean the cluster by taking the following steps:
+7. Load data to TiFlash for analysing:
```
Comment (zanmato1984) [inline quick-start-with-tidb.md:99]: analysing -> analysis

Comment (TisonKun) [reply quick-start-with-tidb.md]: updated

**Comment by TisonKun** [comment]
CI failure seems unrelated to this PR.

**Comment by TomShawn** [comment]
> CI failure seems unrelated to this PR.

We can ignore the failure.

**Comment by TisonKun** [comment]
> > CI failure seems unrelated to this PR.
> 
> We can ignore the failure.

Any further concern blocks this pull request?

**Comment by TomShawn** [comment]
> > > CI failure seems unrelated to this PR.
> > 
> > 
> > We can ignore the failure.
> 
> Any further concern blocks this pull request?

No. @zanmato1984 @superlzs0476 PTAL this PR, thanks!

**Review by zanmato1984 — APPROVED** [review]
LGTM

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -337,6 +341,11 @@ Other requirements for the target machine:
     - [Deploy TiDB offline using TiUP](/production-offline-deployment-using-tiup.md)
     - [Deploy TiDB on Cloud using TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/v1.1)
 
+- If you look for analytics solution with TiFlash:
+
+    - [Use TiFlash](/tiflash/use-tiflash.md)
+    - [TiFlash Overfllow](/tiflash/tiflash-overview.md)
```
Comment (TisonKun) [inline quick-start-with-tidb.md:347]:
```suggestion
    - [TiFlash Overview](/tiflash/tiflash-overview.md)
```

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -96,7 +96,11 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
 
 6. Access the [TiDB Dashboard](/dashboard/dashboard-intro.md) at <http://127.0.0.1:2379/dashboard>. The default username is `root`, with an empty password.
 
-7. After the test, you can clean the cluster by taking the following steps:
+7. Load data to TiFlash for analysis:
+
+    - See also [Use TiFlash](/tiflash/use-tiflash.md).
```
Comment (yikeke) [inline quick-start-with-tidb.md:101]:
```suggestion
7. (Optional) [Load data to TiFlash](/tiflash/use-tiflash.md) for analysis.
```

File: quick-start-with-tidb.md [hunk 4]
```diff
@@ -337,6 +341,11 @@ Other requirements for the target machine:
     - [Deploy TiDB offline using TiUP](/production-offline-deployment-using-tiup.md)
     - [Deploy TiDB on Cloud using TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/v1.1)
 
+- If you look for analytics solution with TiFlash:
```
Comment (yikeke) [inline quick-start-with-tidb.md:344]:
```suggestion
- If you're looking for analytics solution with TiFlash:
```

**Review by yikeke — APPROVED** [review]
rest lgtm

**Comment by TisonKun** [comment]
@yikeke thanks for your suggestions. Applied.

**Pull request merged**